README.md

```diff
@@ -0,0 +1,219 @@
+
+#### Default to trust
+
+Simply put, I'll trust you until you give me a reason not to. I want my direct reports to have a lot of autonomy and that's only possible when paired with trust. You shouldn't find me micromanaging your work, schedule, or decision-making. I'll likely ask you a lot of probing questions but this is generally due to curiosity or a desire to gain more context.
+
+#### Feedback will be delivered in a timely fashion
+
+Giving feedback, both positive and constructive, is one of the most important functions I perform in my role as a manager. Since feedback has such high value, not giving it to you as soon as I have it would be doing you a disservice. You'll often find me giving you feedback right away, in the moment when I perceive something. When it's not possible to provide that realtime feedback or when I come to have feedback outside of direct interactions with you, I will provide this to you in our next 1:1. It's my goal for you to always know how you're doing. If there are surprises for you in a performance feedback loop, it's an indication that I'm not being effective at this.
+
+#### Pointing you in the right direction
+
+When you need it, I'll help get you oriented in the right direction. In the event that I don't know what that is, I'll either identify someone who will or commit to coming back to you with an answer.
+
+### What I will expect of you
+
+#### Give your best in everything you do
+
+If you're going to do something, do it as well as you can. I don't expect that outcomes will be perfect all the time, just that you'll give your best effort. When you find yourself not striving to do your best on something, I encourage you to take a step back and examine why that is. Perhaps it's not an activity you should be spending your time on.
+
+#### Try to get better every day
+
+It's important for you to prioritize your growth. When you become more capable and can rise to new challenges it benefits you (better career opportunities), me (my success is based on your increasing success), and our employer (solving bigger problems, higher productivity, etc). You may not succeed in improving every day but, if that's your goal, it's likely to happen more days than not. Making this type of incremental daily progress leads to noticeable improvement in weeks, a significant change in months, and a radical difference over the course of a year. Get a little bit better every day and don't ever quit. That's the path to greatness.
+
+#### Act like an owner
+
+Treat our software, team, Engineering organization, and company like you own it. If you've got equity, the latter will be literally true, but this means more than that. It's about acting like an owner and caring about what goes on in a way that you would if it was entirely yours. When you see a problem, call it out. If you've got a solution, suggest it. Help us push forward on more fronts than just building software.
+
```

Comment:
This helped me out a ton and is exellcent information for new hires!  One conversation that we had that really stuck out to me was on the topic of Ownership and Fearlessness. You had mentioned that you like to group those two cultural values together because, it emphasizes the importance of not being afraid to give your professional opinion about making company better; whether product, culture, process..etc.

Reply:
One thing that I really liked when I worked at Modcloth is that our team held quarterly brainstorming meetings where everyone shared their ideas for improving the product, and typically a non-technical team within the org was also invited to this session as well.  The ideas were openly discussed, refined, and evaluated by the PMs and many times immediately incorporated into the product feature pipeline.  I felt like these sessions were very ownership-empowering as many engineers got to see their ideas get incorporated into the product.  I'm just sharing this as I think there is more that we can do organizationally at Optimizely to cultivate a sense of ownership.  I know the hackathons fill this role in a way, but I think there are other ways to encourage this in an organization.

Reply:
@kwalker3690 I think you're right that it shouldn't be. I often capitalize Engineering when I'm referring to the organization as an entity, which I think is correct. But, in this case, including 'organization' makes that non-applicable. 👍